src/lender/admin/pool.sol

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2020 Centrifuge
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.15 <0.6.0;
+
+import "tinlake-auth/auth.sol";
+
+interface AssessorLike {
+    function file(bytes32 name, uint256 value) external;
+}
+
+interface LendingAdapter {
+    function raise(uint256 amount) external;
+    function sink(uint256 amount) external;
+}
+
+contract MemberlistLike {
+    function updateMember(address usr, uint256 validUntil) public;
+    function updateMembers(address[] memory users, uint256 validUntil) public;
+}
+
+// Wrapper contract for various pool management tasks.
+contract PoolAdmin is Auth {
+  
+    AssessorLike public assessor;
+    LendingAdapter public lending;
```

Comment:
Should we call this maker? will we have more than once instance? What about aave? Will we need this for aave too?

Reply:
I based this on how it's called in the reserve: https://github.com/centrifuge/tinlake/blob/develop/src/lender/reserve.sol

Reply:
I'm not sure whether we're going to need it for aave, but I think keeping it a bit generic can't hurt either way. I think it's unlikely that we'll have more than one instance right? Since if we had a pool that was integrated both with Maker and Aave, we would probably create a "wrapper lending adapter" which links to the separate lending adapters.